### PR TITLE
Updated Dockerfile to 1.11.1-alpine

### PR DESCRIPTION
**Pull request by gcstang**
Update proxy Dockerfile as per https://github.com/gomods/athens/issues/765

### CODE REVIEW

File: cmd/proxy/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # https://github.com/docker-library/buildpack-deps/blob/1845b3f918f69b4c97912b0d4d68a5658458e84f/stretch/scm/Dockerfile
 # https://github.com/golang/go/blob/f082dbfd4f23b0c95ee1de5c2b091dad2ff6d930/src/cmd/go/internal/get/vcs.go#L90
 
-FROM golang:1.11-alpine AS builder
+FROM golang:1.11.1-alpine AS builder
```
Comment (manugupt1) [inline cmd/proxy/Dockerfile]: Does not golang:1.11-alpine share tag with golang:1.11.1-alpine

Comment (robjloranger) [reply cmd/proxy/Dockerfile]: nice catch, yes it does.

Comment (gcstang) [reply cmd/proxy/Dockerfile]: oops, missed that.

Comment (robjloranger) [reply cmd/proxy/Dockerfile]: I guess we can just close this then.

**Comment by robjloranger** [comment]
closing per discussion above

**Pull request closed without merging**